packages/components/src/border-control/border-control/hook.ts

```diff
@@ -56,7 +56,7 @@ export function useBorderControl(
 
 			onChange( newBorder );
 		},
-		[ onChange, shouldSanitizeBorder, sanitizeBorder ]
```

Comment:
@aaronrobertshaw , do you remember if there's any particular reason why you added `sanitizeBorder` to the list of dependencies here?

Reply:
🤔 Honestly can't recall and looks like I added it before tidying up local experimentation and drafting the initial PR.

I probably just had it in my head that changing `sanitizeBorder` would mean a changed callback. It makes sense that coming from an outer scope it isn't a valid dependency and should be removed.